pdfbox/src/main/java/org/apache/pdfbox/rendering/PDFRenderer.java

```diff
@@ -140,6 +151,11 @@ public BufferedImage renderImage(int pageIndex) throws IOException
         return renderImage(pageIndex, 1);
     }
 
+    public BufferedImage renderImage(PDPage page) throws IOException

```

Comment:
I'm just trying to maintain a similar number of options as the original PDFBox provides, with the added ability to pass in a PDPage object instead of just passing an index.